bucket.go

```diff
@@ -20,6 +20,8 @@ type Bucket interface {
 	GetName() string
 	Get(k string, rv interface{}) error
 	GetRaw(k string) ([]byte, error)
+	Gets(k string, rv interface{}, caso *uint64) error
```

Comment:
Yeah, I would do it in sync-gateway/master and then rebase any features branches that need this change off of master